docs/update-media-paths.md

```diff
@@ -0,0 +1,40 @@
+# TubeSync
+
+## Advanced usage guide - update media paths from the command line
+
+This command updates all media paths with what is current in the database.
+This may want to be done when the Media Format has been changed for a source,
+especially if the folder structure has changed.
+
+
+## Requirements
+
+At least one source and media is present
+
+## Steps
+
+### 1. Run the update media paths command
+
+Execute the following Django command:
+
+`./manage.py update-media-paths`
+
+When deploying TubeSync inside a container, you can execute this with:
+
+`docker exec -ti tubesync python3 /app/manage.py update-media-paths`
+
+There are options for this command:
+* --source
+  * Only target a single source
+  * Use: `./manage.py update-media-paths --source=source_name_here`
+* --no-rename
+  * Do not rename files, only move them to another directory
+  * No value
+  * Use: `./manage.py update-media-paths --no-rename`
+* --dryrun
+  * Make no changes, only print out
+  * No value
```

Comment:
Why state "No value" here when you didn't describe the value with `--source` above?